src/formatDate.js

```diff
@@ -50,7 +53,68 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const prefixYear = (yearBefore, newFormat) => {
+    const yearUpdateFormat
+      = newFormat.find(partDate => partDate.includes('YY'));
+
+    const previousCentury = '19';
+    const currentCentury = '20';
+    const toDetermineCentury = 30;
+
+    if (yearBefore.length === yearUpdateFormat.length) {
+      return yearBefore;
+    }
+
+    if (yearBefore.toString().length > yearUpdateFormat.length) {
+      return yearBefore.slice(-2);
+    }
+
+    if (yearUpdateFormat.length === 4 && yearBefore >= toDetermineCentury) {
+      return previousCentury + yearBefore;
+    }
+
+    if (yearUpdateFormat.length === 4 && yearBefore < toDetermineCentury) {
+      return currentCentury + yearBefore;
+    }
+
+    return yearBefore;
```

Comment:
You work out all the possible conditions for your values and this `return` will never work, are you sure you need it here?